### PR TITLE
feat(mojaloop/3089): changed sdk scheme adapter chart version to mojaloop version

### PR DESCRIPTION
**Pull request by vijayg10**

### CODE REVIEW

File: mojaloop-payment-manager/Chart.yaml [hunk 1]
```diff
@@ -25,12 +25,13 @@ dependencies:
     alias: management-api
```
Comment (mdebarros) [inline mojaloop-payment-manager/Chart.yaml:25]: Please bump chart version.

**Review by mdebarros — COMMENTED** [review]
Lgtm, just one minor comment.

File: mojaloop-payment-manager/Chart.yaml (hunk 1 above)
Comment (vijayg10) [reply mojaloop-payment-manager/Chart.yaml:25]: Thanks @mdebarros , I bumped up the major version as this is a braking change. Old helm values dont work with this version.

Comment (vijayg10) [reply mojaloop-payment-manager/Chart.yaml:25]: And helm upgrade worked with new values

**Review by mdebarros — COMMENTED** [review]
Happy to approve once CI tests pass.

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**